src/Ampersand/Output/FSpec2Pandoc.hs

```diff
@@ -135,7 +135,10 @@ fSpec2Pandoc env now fSpec = (thePandoc, thePictures)
         . mconcat
         $ blocksByChapter
 
-    thePictures = concat picturesByChapter
+    thePictures = map (makePicture env fSpec) largePictures ++ concat picturesByChapter
+      where
+        largePictures = [PTClassDiagram, PTLogicalDM True, PTLogicalDM False, PTTechnicalDM]
+
```

Comment:
These 'largePictures' should come from the individual chapters (use picturesByChapter). Now the feature that the user can specify what chapters she wants to have (and which not) is probably broken.

Reply:
The large pictures are extras. They are not included in a chapter. The large pictures are therefore generated irrespective of the choice of chapters.